src/c++/vdo/base/dm-vdo-target.c

```diff
@@ -2387,11 +2397,16 @@ static void load_callback(struct vdo_completion *completion)
 		vdo_allow_read_only_mode_entry(completion);
 		return;
 
+#if defined(VDO_INTERNAL) || defined(INTERNAL)
 	case LOAD_PHASE_STATS:
 		vdo_continue_completion(completion, vdo_initialize_kobjects(vdo));
 		return;
 
+#endif
 	case LOAD_PHASE_LOAD_DEPOT:
+#if !defined(VDO_INTERNAL) && !defined(INTERNAL)
+		vdo_set_dedupe_state_normal(vdo->hash_zones);
+#endif
 		if (vdo_is_read_only(vdo)) {
 			/*
```

Comment:
Here. Can we just call vdo_set_dedupe_state_normal() unconditionally here?

Reply:
I think so, yes; trying it out...